Makefile

```diff
@@ -3,47 +3,56 @@ build-cli:
 	cargo build -p anchor-cli --release
 	cp target/release/anchor cli/npm-package/anchor
 
+.PHONY: build-tests-bpf-%
```

Comment:
Why are makefile changes needed? Seems like they shouldn't be.

Reply:
On the current makefile rules build-example-bpf-all do not work because basic-0 to basic-4 are not located on "examples/" but in "examples/tutorial" (and basic-5 do not exist anymore)
Also all other examples are not located on "examples/" but on "tests/".
Since tests and tutorial is now split in two different folder I separate them in 2 different rule: "build-tests-bpf-all" and "build-example-bpf-all".
I added the rule "build-all" to build tests and tutorials.